oden_print.sh

```diff
@@ -1,19 +1,45 @@
-function oden_print {
+unction oden_print {
     # print to an Oden printer wirelessly via SSH to workstation
     # first argument: printer name
     # other arguments: files to be printed 
```

Comment:
```suggestion
```
Let's remove these. We already have to keep documentation updated in multiple places, and it has help flag built in.